packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/index.ts

```diff
@@ -69,7 +69,10 @@ export {
   Origin,
   KnownActionType,
   ActionType,
-  Versions,
+} from "./models/index.js";
+export {
+  NetworkAnalyticsContext,
```

Comment:
why we export NetworkAnalyticsContext to public in classical layer? I think this is implementation detail in this layer.

Reply:
fixed in https://github.com/Azure/autorest.typescript/pull/2759/commits/219814be49d41480c8d245e3402240c4c81ebbeb